contents/articles/2016-11_focal-musical-expression/index.md

```diff
@@ -0,0 +1,37 @@
+--- 
+title: An Eye-Tracking Musical Expression Controller
+date: Fri Nov 11 2016 14:43:37 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Focal is an experimental concept that uses AR headset with eye tracking as a digital musical controller..."
```

Comment:
grammar and copy edit: 
... that uses an AR headset with Pupil Labs eye tracking add-ons...